src/viam/components/arm/arm.py

```diff
@@ -1,10 +1,10 @@
 import abc
-from typing import Any, Dict, Final, Optional, Tuple
+from typing import Any, Dict, Final, List, Optional, Tuple
 
 from viam.resource.types import RESOURCE_NAMESPACE_RDK, RESOURCE_TYPE_COMPONENT, Subtype
 
 from ..component_base import ComponentBase
-from . import JointPositions, KinematicsFileFormat, Pose
+from . import Geometry, JointPositions, KinematicsFileFormat, Pose
 
 
 class Arm(ComponentBase):
```

Comment:
Q: it seems like we are adding `get_geometries` to every component interface - should we add this to `ComponentBase` then?